Image.php

```diff
@@ -674,6 +683,10 @@ public function save($file, $type = 'guess', $quality = 80)
                 $success = $this->getAdapter()->savePng($file);
             }
 
+            if ($type == 'webp') {
```

Comment:
Please use strict comparison.

Reply:
Would love to, but the rest of the checks don't either, and I went for code consistency, imho this is for another PR, one that cleans up the code base

Reply:
@Soullivaneuh honestly. I think your being overly critial in these reviews considering the codebase for Gregwar doesn't follow strict standards and it's barely even maintained by it's owner anymore. I would be thankful for any collaboration from maintainers.

Reply:
@Soullivaneuh I am ok to merge and fix this in another PR, along with code standard if needed